CMakeLists.txt

```diff
@@ -78,7 +78,7 @@ if(XEVD_APP_STATIC_BUILD)
 endif(XEVD_APP_STATIC_BUILD)
 
 cmake_policy(SET CMP0048 NEW)
-set(CMAKE_C_STANDARD 99)
```

Comment:
For now we want to keep C99

Reply:
I'm not sure why Github isn't updating this PR. I rebased and force-pushed. This should be at 2ebfd0794b92c5f5a2bf9b1220969dca509176fb.

Reply:
Welp, I'm an idiot. Confusing this with the PR on xeve. I'll give this one the same treatment tomorrow.

Reply:
Rebased and dropped everything that's not strictly related to Darwin, opened separate PRs for those changes.